packages/exporter-trace-otlp-http/test/node/CollectorTraceExporter.test.ts

```diff
@@ -354,4 +409,44 @@ describe('OTLPTraceExporter - node with json over http', () => {
       });
     });
   });
+  describe('export - with timeout', () => {
+    beforeEach(() => {
+      fakeRequest = new Stream.PassThrough();
+      stubRequest = sinon.stub(http, 'request').returns(fakeRequest as any);
```

Comment:
Not sure if it is possible in CI but I think some test using a real HTTP request would be good. Currently it seems we just stub HTTP so we might miss some HTTP special stuff.
There might be different behavior at which stage of the HTTP request the timeout happens (socket creation, sending request, receiving response,...).
There might be also differences between node versions.

Reply:
Has this been addressed?

Reply:
I think it is ok here. Since the timer is created manually (https://github.com/open-telemetry/opentelemetry-js/pull/2738/files#diff-d5312ed9e76f7794f25948cf9ac39d6ee5103761f551fef74f8c154362e01d94R49) instead of the socket timer, it is just a timer that verifies that the request ends in a specific duration, otherwise we abort the request and log the event.